src/api/index.tsx

```diff
@@ -151,3 +155,25 @@ export const getCurrentUser = async (apiState: ApiState) => {
     endpoint: '/accounts/profile/',
   });
 };
+
+type GetDiffParams = {
+  addonId: number;
+  apiState: ApiState;
+  baseVersionId: number;
+  headVersionId: number;
+  path?: string;
+};
+
+export const getDiff = async ({
+  addonId,
+  apiState,
+  baseVersionId,
+  headVersionId,
+  path,
+}: GetDiffParams) => {
+  return callApi<ExternalVersionWithDiff>({
+    apiState,
+    endpoint: `reviewers/addon/${addonId}/versions/${baseVersionId}/compare_to/${headVersionId}`,
+    query: path ? { file: path } : undefined,
```

Comment:
When would we not have a path? I guess this is the default screen, where no file has been chosen yet (like with the Browse page)?

Reply:
Yep, exactly. This is very similar to the Browse API endpoint. There is a default file.